readme.md

```diff
@@ -75,6 +75,13 @@ Default: `true`
 
 [URL encode](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) the keys and values.
 
+#### arrayFormat
+
+Type: `string`<br>
+Default: `'none'`
+
+Supports both `index` for an indexed array representation or `bracket` for a _bracketed_ array representation.
```

Comment:
Show examples of what the different `arrayFormat`'s results in. Not clear for the user based on just this text.